### ISSUE TITLE
Home page - keyboard focus not visible for app store icons

### ISSUE THREAD

**Issue by BNewing** [zooniverse/Panoptes-Front-End#4105]
## Expected behavior
When tabbing through the home page, you should be able to see keyboard focus outlining on the app store icons.

## Current behavior
No highlighting is obviously visible, as demonstrated here:
<img width="468" alt="screen shot 2017-10-19 at 17 32 37" src="https://user-images.githubusercontent.com/16868713/31782718-c02198f8-b4f3-11e7-88f2-05ac91df07e9.png">

Looking through the styling, it looks like it's caused by this:

<img width="324" alt="screen shot 2017-10-19 at 17 31 06" src="https://user-images.githubusercontent.com/16868713/31782732-d114d364-b4f3-11e7-93ad-e35472e36dc6.png">

## Steps to replicate
Open up the home page and keep tabbing down until you reach the app store icons (there are other tabbing issues that you'll see in the process but they're separate issues)

## Additional information

- **Operating system:**
Mac OS
- **Browser:**
Chrome

**Comment by BNewing** [comment]
This ticket references app store icons in particular, but the CSS outline being removed causes issues across all content that isn't a button but is still keyboard accessible, for example the social media icons too

**Comment by shoemakerdr** [comment]
I'd like to help. A lot of people get rid of outlines because native outlines clash with the styling. Would you want to just universally allow the default outline throughout the app or would you want outline styling specific to Zooniverse?

**Comment by BNewing** [comment]
Hurrah! :D 

I'm not sure - I would un-remove the outlining from the homepage and restyle it so it looked a bit more like something off of the gov.uk website, because they're pretty good generally. I'd then investigate everything else to see where else it has been removed/taken off/to make it more consistent across the site :)

**Comment by beckyrother** [comment]
Hi @BNewing and @shoemakerdr ! Thank you for bringing this to our attention. 

It looks like the current behavior applies the focus color to buttons, but you're right, definitely should not apply `outline: none`. Since the rest of the site uses the browser-default outline, let's continue to do that, rather than create custom styling.

**Comment by shoemakerdr** [comment]
Got it. I'll submit a PR.

**Comment by shoemakerdr** [comment]
I think I see why they took the outline out. It looks really bad in Chrome:

![zoo-outline-chrome](https://user-images.githubusercontent.com/24695425/34536138-df5eaaf4-f089-11e7-9120-ce8ac1abbaef.jpeg)

Though it looks totally fine in FireFox:

![zoo-outline-firefox](https://user-images.githubusercontent.com/24695425/34536146-e6c2d31a-f089-11e7-8da9-37ac5f06f8a4.jpeg)

If we change the a tag to have a display of inline-block, that fixes this weird looking behavior but has extra margin:

![zoo-outline-chrome-fix](https://user-images.githubusercontent.com/24695425/34536347-6d007a40-f08a-11e7-84a5-b9681040cbc2.jpeg)

I could take the margin styling out of the child imgs and add that to the parent a tag instead, which would look a lot cleaner:

![zoo-outline-fix2](https://user-images.githubusercontent.com/24695425/34536547-0f96e492-f08b-11e7-9fd1-5a6166381ef1.jpeg)

There's some other elements that are nested similarly:

![zoo-outline-chrome2](https://user-images.githubusercontent.com/24695425/34536576-285787de-f08b-11e7-95a3-df1c6bb4e478.jpeg)

Should I go ahead and take care of these?
